unlock-app/src/components/interface/verification/MembershipCard.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  AvatarImage,
+  Fallback as AvatarFallback,
+  Root as Avatar,
+} from '@radix-ui/react-avatar'
+import { Lock } from '~/unlockTypes'
+import dayjs from 'dayjs'
+import relativeTimePlugin from 'dayjs/plugin/relativeTime'
+import { addressMinify } from '~/utils/strings'
+import { useConfig } from '~/utils/withConfig'
+import {
+  RiCloseCircleFill as InvalidIcon,
+  RiCheckboxCircleFill as ValidIcon,
+} from 'react-icons/ri'
+import { ReactNode } from 'react'
+
+dayjs.extend(relativeTimePlugin)
+
+interface Props {
+  timestamp: number
+  lock: Lock
+  membershipData: any
```

Comment:
```suggestion
  membershipData: {
  expiration: number
    image?: string
    keyId?: string | number
    owner?: string
    userMetadata?: {
      public?: any
      protected?: any
    }
  }
```

Reply:
Will add this as a separate type so it's reusable. Thanks.